_data-prepper/pipelines/configuration/processors/otel-trace-group.md

```diff
@@ -0,0 +1,107 @@
+
+```
+pipeline:
+  ...
+  processor:
+    - otel_trace_group:
+        hosts: ["https://your-amazon-opensearch-service-endpoint"]
+        aws_sigv4: true
+        cert: path/to/cert
+        insecure: false
+```
+
+See [OpenSearch Sink Security](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/security.md) for a detailed explanation.
+
+## Configuration
+
+You can configure the Amazon OpenSearch Service with the following options.
+
+| Name | Required | Description | Default value
+| -----| ----| -----------| -----|
+|`hosts`| Yes | A list of IP addresses of OpenSearch nodes.
+|`cert` | No | A certificate authority (CA) certificate that is PEM-encoded. Accepts both .pem or .crt. This enables the client to trust the CA that has signed the certificate that OpenSearch is using. | `null` |
+|`aws_sigv4` | Yes | A boolean flag to sign the HTTP request with AWS credentials. Only applies to Amazon OpenSearch Service. See [OpenSearch security](https://github.com/opensearch-project/data-prepper/blob/129524227779ee35a327c27c3098d550d7256df1/data-prepper-plugins/opensearch/security.md) for details. | `false`. |
+|`aws_region` | A String that represents the region of Amazon OpenSearch Service domain, for example, `us-west-2`. Only applies to Amazon OpenSearch Service. | `us-east-1` |
+|`aws_sts_role_arn`| An identity and access management (IAM) role that the sink plugin assumes to sign the request to the Amazon OpenSearch Service. If not provided, the plugin uses the default credentials. <!--- Should those credentials be specified here?--->
+| `aws_sts_header_overrides` | No | A map of header overrides to make when assuming the IAM role for the sink plugin. |
+| `insecure` | Yes | A boolean flag to turn off SSL certificate verification. If set to `true`, CA certificate verification is turned off and insecure HTTP requests are sent. | `false` |
+| `username` | Yes | A string that contains the username and is used in the [internal users](https://opensearch.org/docs/latest/security/access-control/users-roles/) of your OpenSearch cluster. | `null` |
+| `password` | No | A string that contains the password and is used in the [internal users](https://opensearch.org/docs/latest/security/access-control/users-roles/) of your OpenSearch cluster. | `null` |
+
+## Metrics
+
+The following table describes metrics that are common to all processors.
+
+| Metric name | Type | Description |
+| ------------- | ---- | -----------|
+| `recordsIn` | Counter | Metric representing the ingress of records to a pipeline component. |
+| `recordsOut` | Counter | Metric representing the egress of records from a pipeline component. |
+| `timeElapsed` | Timer | Metric representing the time elapsed during execution of a pipeline component. | 
+
+
+### Counter
+
+The `otel_trace_group` processor includes the following custom counter metrics.
+
+| Metric name | Description |
```

Comment:
Rather than have a `### Counter` header, I think it would be ideal to keep the type column, just as you do with the other metrics.


```
## Metrics

This plugin provides the following metrics.

 | Metric name | Type | Description |
 | ------------- | ---- | -----------|
 | `recordsIn` | Counter | Metric representing the ingress of records to a pipeline component. |
 | `recordsOut` | Counter | Metric representing the egress of records from a pipeline component. |
 | `timeElapsed` | Timer | Metric representing the time elapsed during execution of a pipeline component. | 
 | `recordsInMissingTraceGroup ` | Counter | The number of ingress records missing trace group fields. |
 | `recordsOutFixedTraceGroup ` | Counter | The number of egress records with successfully completed trace group fields. |
 | `recordsOutMissingTraceGroup ` | Counter | The number of egress records missing trace group fields. |
```

Reply:
@dlvenable This looks great. Can you provide the `type` for `recordsInMissingTraceGroup`, `recordsOutFixedTraceGroup`, and `recordsOutMissingTraceGroup`?